tools/test-npm.sh

```diff
@@ -29,10 +29,10 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-
-../$NODE cli.js install --ignore-scripts
-../$NODE cli.js run-script test-legacy
-../$NODE cli.js run-script test
+NODEPATH="$(../$NODE -p 'require("path").resolve("..")')"
```

Comment:
@rvagg can I get a quick LGTM on changing this line to `export PATH="$(../$NODE -p 'require("path").resolve("..")'):$PATH"`?

That will mean we no longer require the `PATH="$NODEPATH:$PATH"` on the 3 lines below.